docs/Server-Methods.md

```diff
@@ -6,6 +6,17 @@
 #### server
 `fastify.server`: The Node core [server](https://nodejs.org/api/http.html#http_class_http_server) object as returned by the [**`Fastify factory function`**](https://github.com/fastify/fastify/blob/master/docs/Factory.md).
 
+<a name="after"></a>
+### after
+Invoked when the current plugin and all the plugin that has been registered under
+him have finished loading and is executed before the method ``` fastify.ready ```
+
+```js
+fastify.after(err => {
+  console.log('this plugin is loaded!')
+})
+```
```

Comment:
I think you should add also `register` and `ready`, in this way the user can try it and it will be more clear.
```js
fastify
  .register((instance. opts, next) => {
    console.log('Current plugin')
    next()
  })
  .after(err => {
    console.log('After current plugin')
  })
  .register((instance. opts, next) => {
    console.log('Next plugin')
    next()
  })
  .ready(err => {
    console.log('Everything has been loaded')
  })
```